### ISSUE TITLE
NoBlackSky errors on 1.16.X

### ISSUE THREAD

**Issue by HyperKids** [gamerover98/NoBlackSky#1]
```
>.... [02:35:55 ERROR]: [NoBlackSky] Unhandled exception occured in onPacketSending(PacketEvent) for NoBlackSky
java.lang.IllegalArgumentException: Unable to find a method that matches {modifiers=[required: 0, banned: 0], params=[{Type: class name of net\.minecraft\.(\p{javaJavaIdentifierStart}\p{javaJavaIdentifierPart}*\.)+\p{javaJavaIdentifierStart}\p{javaJavaIdentifierPart}*, Index: null}, {Type: Exact class java.lang.String, Index: 0}, {Type: Exact class java.lang.String, Index: 1}, {Type: Exact class java.lang.String, Index: 4}], paramCount=5}
at com.comphenix.protocol.reflect.FuzzyReflection.getMethod(FuzzyReflection.java:174) ~[?:?]
at com.comphenix.protocol.utility.MinecraftReflection.getWorldTypeClass(MinecraftReflection.java:1173) ~[?:?]
at com.comphenix.protocol.events.PacketContainer.getWorldTypeModifier(PacketContainer.java:392) ~[?:?]
at it.gamerover.nbs.wrapper.WrapperPlayServerLogin.setLevelType(WrapperPlayServerLogin.java:181) ~[?:?]
at it.gamerover.nbs.NBS_PacketAdapter.onPacketSending(NBS_PacketAdapter.java:81) ~[?:?]
at com.comphenix.protocol.injector.SortedPacketListenerList.invokeSendingListener(SortedPacketListenerList.java:195) ~[?:?]
at com.comphenix.protocol.injector.SortedPacketListenerList.invokePacketSending(SortedPacketListenerList.java:149) ~[?:?]
at com.comphenix.protocol.injector.PacketFilterManager.handlePacket(PacketFilterManager.java:588) ~[?:?]
at com.comphenix.protocol.injector.PacketFilterManager.invokePacketSending(PacketFilterManager.java:564) ~[?:?]
at com.comphenix.protocol.injector.netty.ProtocolInjector.packetQueued(ProtocolInjector.java:338) ~[?:?]
at com.comphenix.protocol.injector.netty.ProtocolInjector.onPacketSending(ProtocolInjector.java:298) ~[?:?]
at com.comphenix.protocol.injector.netty.ChannelInjector.processSending(ChannelInjector.java:378) ~[?:?]
at com.comphenix.protocol.injector.netty.ChannelInjector.access$800(ChannelInjector.java:64) ~[?:?]
at com.comphenix.protocol.injector.netty.ChannelInjector$3.handleScheduled(ChannelInjector.java:343) ~[?:?]
at com.comphenix.protocol.injector.netty.ChannelInjector$3.onMessageScheduled(ChannelInjector.java:313) ~[?:?]
at com.comphenix.protocol.injector.netty.ChannelProxy$2.schedulingRunnable(ChannelProxy.java:127) ~[?:?]
at com.comphenix.protocol.injector.netty.EventLoopProxy.execute(EventLoopProxy.java:95) ~[?:?]
at net.minecraft.server.v1_16_R2.NetworkManager.b(NetworkManager.java:288) ~[patched_1.16.3.jar:git-Paper-220]
at net.minecraft.server.v1_16_R2.NetworkManager.dispatchPacket(NetworkManager.java:244) ~[patched_1.16.3.jar:git-Paper-220]
at net.minecraft.server.v1_16_R2.NetworkManager.sendPacket(NetworkManager.java:220) ~[patched_1.16.3.jar:git-Paper-220]
at net.minecraft.server.v1_16_R2.PlayerConnection.a(PlayerConnection.java:1630) ~[patched_1.16.3.jar:git-Paper-220]
at net.minecraft.server.v1_16_R2.PlayerConnection.sendPacket(PlayerConnection.java:1603) ~[patched_1.16.3.jar:git-Paper-220]
at net.minecraft.server.v1_16_R2.PlayerList.a(PlayerList.java:189) ~[patched_1.16.3.jar:git-Paper-220]
at net.minecraft.server.v1_16_R2.LoginListener.c(LoginListener.java:167) ~[patched_1.16.3.jar:git-Paper-220]
at net.minecraft.server.v1_16_R2.LoginListener.tick(LoginListener.java:65) ~[patched_1.16.3.jar:git-Paper-220]
at net.minecraft.server.v1_16_R2.NetworkManager.a(NetworkManager.java:380) ~[patched_1.16.3.jar:git-Paper-220]
at net.minecraft.server.v1_16_R2.ServerConnection.c(ServerConnection.java:145) ~[patched_1.16.3.jar:git-Paper-220]
at net.minecraft.server.v1_16_R2.MinecraftServer.b(MinecraftServer.java:1383) ~[patched_1.16.3.jar:git-Paper-220]
at net.minecraft.server.v1_16_R2.DedicatedServer.b(DedicatedServer.java:371) ~[patched_1.16.3.jar:git-Paper-220]
at net.minecraft.server.v1_16_R2.MinecraftServer.a(MinecraftServer.java:1211) ~[patched_1.16.3.jar:git-Paper-220]
at net.minecraft.server.v1_16_R2.MinecraftServer.w(MinecraftServer.java:999) ~[patched_1.16.3.jar:git-Paper-220]
at net.minecraft.server.v1_16_R2.MinecraftServer.lambda$a$0(MinecraftServer.java:177) ~[patched_1.16.3.jar:git-Paper-220]
at java.lang.Thread.run(Thread.java:748) [?:1.8.0_242]
>.... [02:35:55 ERROR]: Parameters:
net.minecraft.server.v1_16_R2.PacketPlayOutLogin@596a96d7[
a=996
b=8627381720829052060
c=false
d=SURVIVAL
e=NOT_SET
f=[ResourceKey[minecraft:dimension / minecraft:overworld], ResourceKey[minecraft:dimension / minecraft:fsab]]
g=net.minecraft.server.v1_16_R2.IRegistryCustom$Dimension@1c9b4ab9
h=net.minecraft.server.v1_16_R2.DimensionManager@1184b542
i=ResourceKey[minecraft:dimension / minecraft:overworld]
j=300
k=10
l=false
m=true
n=false
o=false
```

**Comment by gamerover98** [comment]
Hi, this is a knowed issue. I don't know why Tiny Protocol (a ProtocolLib small interface) fails when it tries to edit the packet. I will check.

**Comment by gamerover98** [comment]
The plugin can't run on 1.16 due to a ProtocolLib development build issue.
I've reported the bug to its GitHub page: https://github.com/dmulloy2/ProtocolLib/issues/1009

Let's wait till a stable version is released...

**Comment by gamerover98** [comment]
Fixed with the commit b5600b6f5b89577bf0fccccdc74d7c7387cc5024!

You will be able to download the latest version of NoBlackSky plugin on [SpigotMC](https://www.spigotmc.org/resources/no-black-sky-1-7-x-1-16-x-for-void-worlds.25337/)
